quantum/snippets.js

```diff
@@ -87,6 +87,25 @@ const CIRCUIT_DIAGRAM =
 `qc.draw(output='mpl')
 `;
 
+const GROVERS_ALGO =
+`from qiskit import QuantumCircuit
+from qiskit import Aer
+from qiskit.utils import QuantumInstance
+from qiskit.quantum_info import Statevector
+from qiskit.algorithms import Grover, AmplificationProblem
+
+element = '%s'
+oracle = Statevector.from_label(element)
+problem = AmplificationProblem(oracle=oracle, is_good_state = lambda bitstr: bitstr==element)
+backend = Aer.get_backend('qasm_simulator')
+grover = Grover(quantum_instance=backend)
+result = grover.amplify(problem)
+print('Success!' if result.oracle_evaluation else 'Failure!')
```

Comment:
What is the purpose of this print, given that it doesn't have any effect on the rest of the output? (e.g. it outputs "success" or "failure", but nothing happens as a result outside of the print).

Reply:
Yes, I can remove it, it was for rather for debug purpose only.